place_tracker/lib/place_map.dart

```diff
@@ -0,0 +1,441 @@
+import 'dart:math';
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+import 'package:google_maps_flutter/google_maps_flutter.dart';
+
+import 'place.dart';
+import 'place_stub_data.dart';
+import 'place_details.dart';
+
+class PlaceMap extends StatefulWidget {
+  const PlaceMap({
+    Key key,
+    this.center,
+  }) : super(key: key);
+
+  final LatLng center;
+
+  @override
+  PlaceMapState createState() => PlaceMapState();
+}
+
+class PlaceMapState extends State<PlaceMap> {
+
+  static BitmapDescriptor _getPlaceMarkerIcon(PlaceCategory category) {
+    // TODO(kenzieschmoll): use custom marker assets.
+    double markerHue;
+    switch (category) {
+      case PlaceCategory.favorite:
+        markerHue = BitmapDescriptor.hueRed;
+        break;
+      case PlaceCategory.visited:
+        markerHue = BitmapDescriptor.hueViolet;
+        break;
+      case PlaceCategory.wantToGo:
+      default:
+        markerHue = BitmapDescriptor.hueAzure;
+    }
+    return BitmapDescriptor.defaultMarkerWithHue(markerHue);
+  }
+
+  static List<Place> _getPlacesForCategory(
```

Comment:
I don't think the line breaks really help here.  
```
static List<Place> _getPlacesForCategory(PlaceCategory category, Map<Marker, Place> places) {
  return places.values.where((Place place) => place.category == category).toList();
}
```

Reply:
If I don't use line breaks, the line length will go over the suggested 80 char limit.

Reply:
https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#prefer-line-length-of-80-characters

Although 80 character lines are preferred, if you look at the flutter sources you'll see that we often exceed that limit.

Reply:
Fixed.